mobile/lib/blocs/kya/kya_bloc.dart

```diff
@@ -11,10 +11,23 @@ part 'kya_event.dart';
 part 'kya_state.dart';
 
 class KyaBloc extends HydratedBloc<KyaEvent, KyaState> {
-  KyaBloc() : super(const KyaState(lessons: [])) {
+  KyaBloc() : super(const KyaState(lessons: [], quizzes: [])) {
     on<UpdateKyaProgress>(_onUpdateKyaProgress);
     on<ClearKya>(_onClearKya);
     on<FetchKya>(_onFetchKya);
+    on<FetchQuizzes>(_onFetchQuizzes);
+    on<UpdateQuizProgress>(_onUpdateQuizProgress);
+    on<ClearQuizzes>(_onClearQuizzes as EventHandler<ClearQuizzes, KyaState>);
```

Comment:
We can fix `on<ClearQuizzes>(_onClearQuizzes as EventHandler<ClearQuizzes, KyaState>);` to 
`on<ClearQuizzes>(_onClearQuizzes);`